aeon/dj_pipeline/analysis/visit_analysis.py

```diff
@@ -378,9 +380,9 @@ def make(self, key):
                     "visit_date": visit_date.date(),
                     "day_duration": day_duration,
                     "time_fraction_in_corridor": in_corridor.mean(),
-                    "in_corridor": in_corridor.values,
+                    "in_corridor": np.array(in_corridor.index[np.where(in_corridor)]),
                     "time_fraction_in_arena": in_arena.mean(),
-                    "in_arena": in_arena.values,
+                    "in_arena": np.array(in_arena.index[np.where(in_arena)]),
```

Comment:
```suggestion
                    "in_arena": in_arena.index.values[in_arena],
```

Reply:
Yes, that seems better. Will accept those changes. Thanks.